app/server_app_adapters.go

```diff
@@ -67,7 +66,7 @@ func (s *Server) RunOldAppInitalization() error {
 	}
 
 	if htmlTemplateWatcher, err := utils.NewHTMLTemplateWatcher("templates"); err != nil {
-		mlog.Error(fmt.Sprintf("Failed to parse server templates %v", err))
+		mlog.Error("Failed to parse server templates ", mlog.Err(err))
```

Comment:
Why do we need the trailing space?

Reply:
@iomodo Thanks for your feedback, I'll work on it. There is actually no reason for the trailing space.

Reply:
@iomodo Feedback has been implemented.